### PR TITLE
[IMP] accounting: tax return

### PR DESCRIPTION
**Pull request by chiaraprattico**
taskid: 3056868

### CODE REVIEW

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Hey Chiara, shitty work! ;) I made a few suggestions, feel free to decline if you don't agree with them. :)

I'm adding a comment here since I can't do it directly in the diff:

Line 54: To configure your tax grids, go to...

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 1]
```diff
@@ -2,34 +2,34 @@
...
+The configuration of the **Tax Return Periodicity** allows Odoo Accounting to compute your tax
+return correctly and also to send you a reminder to never miss a tax return deadline.
+
+To do so, go to :menuselection:`Accounting app --> Configuration --> Settings`, and to the
+:guilabel:`Tax Return Periodicity section`.
 
-To do so, go to :menuselection:`Accounting --> Configuration --> Settings --> Fiscal Periods`,
-and go to the **Tax Return Periodicity** section.
+- :guilabel:`Periodicity`: define here whether you submit your tax return on a monthly or quarterly
+  basis;
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:30]:
```suggestion

```
Gap between the lines

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 2]
```diff
@@ -42,15 +42,16 @@ and go to the **Tax Return Periodicity** section.
 Tax Grids
 ---------
 
-Odoo generates Tax Reports based on the **Tax Grids** settings that are configured on your taxes.
-Therefore, it is crucial to make sure that all the recorded transactions use the right taxes. You
-can see on each Journal Item which Tax Grid is used for that transaction.
+Odoo generates tax reports based on the :guilabel:`Tax Grids` settings that are configured on your
+taxes. Therefore, it is crucial to make sure that all recorded transactions use the right taxes.
+You can see the :guilabel:`Tax Grids` by opening any :guilabel:`Invoice` and :guilabel:`Bill` in the
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:47]:
```suggestion
You can see the :guilabel:`Tax Grids` by opening the :guilabel:`Journal Item tab` of any :guilabel:`Invoice` and :guilabel:`Bill`.
```

Comment (toaa-odoo) [reply content/applications/finance/accounting/reporting/declarations/tax_returns.rst:47]: The previous order was confusing/misleading

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 3]
```diff
@@ -42,15 +42,16 @@ and go to the **Tax Return Periodicity** section.
...
+:guilabel:`Journal Item tab`.
```
Comment (toaa-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:48]:
```suggestion

```

**Review by toaa-odoo — CHANGES_REQUESTED** [review]
Very few changes left, but I saw a few lines beyond the 100-character limit ? Is that normal ? :)

**Review by toaa-odoo — APPROVED** [review]

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 4]
```diff
@@ -2,34 +2,34 @@
 Tax return (VAT declaration)
 ============================
 
-Companies that are registered for **VAT (Value Added Tax)** must file a **Tax return** on a monthly
-or quarterly basis, depending on their turnover and the regulation of the country in which they are
-registered. A Tax return - or VAT return - gives the tax authorities information about the taxable
-transactions made by the company, the *output tax* it has charged its customers, and the *input tax*
-its vendors have charged it. Based on these values, the company can calculate the tax amount they
-have to pay or be refunded.
+Companies with a registered for :abbr:`VAT (Value Added Tax)` number must submit a **tax return** on
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:5]:
```suggestion
Companies with a registered :abbr:`VAT (Value Added Tax)` number must submit a **tax return** on
```

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 5]
```diff
@@ -2,34 +2,34 @@
...
+a monthly or quarterly basis, depending on their turnover and the registration regulation. A tax
+return - or VAT return - gives the tax authorities information about the taxable transactions made
+by the company. The **output tax** is charged on the number of goods and services sold by a
+business, while the **input tax** is the tax added to the price when goods or services are
+purchased.
+Based on these values, the company can calculate the tax amount they have to pay or be refunded.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:11]: going to the next line is the same as a space in RST.
```suggestion
purchased. Based on these values, the company can calculate the tax amount they have to pay or be refunded.
```

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 6]
```diff
@@ -2,34 +2,34 @@
...
 
 .. note::
-   You can find information about VAT and its mechanism on `this page from the European Commission
-   <https://ec.europa.eu/taxation_customs/business/vat/what-is-vat_en>`_.
-
-.. todo:: add doc about intracom listing
+   You can find additional information about VAT and its mechanism on this page from the European
+   `"What is VAT?" Commission <https://ec.europa.eu/taxation_customs/business/vat/what-is-vat_en>`_.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:15]: problem with the text and link here

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 7]
```diff
@@ -2,34 +2,34 @@
...
 ----------------------
 
-The configuration of the **Tax Return Periodicity** allows Odoo Accounting to compute your Tax
-Return correctly and also to send you a reminder to never miss a tax return deadline.
+The configuration of the **Tax Return Periodicity** allows Odoo to compute your tax return correctly
+and also to send you a reminder to never miss a tax return deadline.
 
-To do so, go to :menuselection:`Accounting --> Configuration --> Settings --> Fiscal Periods`,
-and go to the **Tax Return Periodicity** section.
+To do so, go to :menuselection:`Accounting --> Configuration --> Settings`.Under the
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:26]:
```suggestion
To do so, go to :menuselection:`Accounting --> Configuration --> Settings`. Under the
```

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 8]
```diff
@@ -59,65 +60,55 @@ grids that are used to record invoices or credit notes.
    :alt: Configure taxes and their tax grids in Odoo Accounting
 
 .. note::
-   Taxes and reports are usually already pre-configured: a *Fiscal Localization Package* is
-   installed according to the country you select at the creation of your database. :doc:`Click here
-   <../../fiscal_localizations/overview/fiscal_localization_packages>` for more information about
-   Fiscal Localization Packages.
-
-.. _tax_return_lock:
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:67]: why did you delete this anchor?

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 9]
```diff
@@ -59,65 +60,55 @@ grids that are used to record invoices or credit notes.
...
+   Taxes and reports are usually already pre-configured in Odoo: a **Fiscal Localization Package**
+   is installed according to the country you select at the creation of your database.
+   For more information, check out the :doc:`Click here <../../fiscal_localizations/overview/fiscal_localization_packages>`
+   for more information about Fiscal Localization Packages.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:66]: you can put a link directly on "fiscal localization package" and write this in sentence case.

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 10]
```diff
@@ -72,52 +73,44 @@ Close a tax period
 Tax Lock Date
 -------------
 
-Any new transaction which accounting date is prior to the **Tax Lock Date** has its tax values moved
-to the next open tax period. This is useful to make sure that no change can be made to a report once
-its period is closed.
+Any new transaction whose accounting date is before to the :guilabel:`Tax Lock Date` has its tax
```
[the remark this answers is not in the document]
Comment (jcs-odoo) [reply content/applications/finance/accounting/reporting/declarations/tax_returns.rst:74]: marked as resolved but it's still there.
Isn't the previous "prior to" actually better?

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst [hunk 11]
```diff
@@ -59,65 +60,55 @@ grids that are used to record invoices or credit notes.
...
-
-After having reviewed the generated Journal Entry, click on *Post*. In addition to posting the entry,
-Odoo automatically creates a PDF file with the **Tax Report** that you can download from the chatter
-and preview on the right column. It includes all the values to report to the tax authorities, along
-with the amount you have to pay or be refunded.
+overview of your tax report.
+From this view, you can easily access different formats of your tax report, such as `PDF` and
+XLSX.
+These include all the values to report to the tax authorities, along with the amount you have to pay
+or be refunded.
```
Comment (jcs-odoo) [inline content/applications/finance/accounting/reporting/declarations/tax_returns.rst:101]: going to the next line doesn't work in RST (also, it's not something usual in English)
either write a space or make it a new paragraph.

**Review by jcs-odoo — CHANGES_REQUESTED** [review]
I skim-read this and left a few comments.

File: content/applications/finance/accounting/reporting/declarations/tax_returns.rst (hunk 8 above)
Comment (chiaraprattico) [reply content/applications/finance/accounting/reporting/declarations/tax_returns.rst:67]: I didn't see the point of it, in this specific point.

**Review by xpl-odoo — APPROVED** [review]

**Review by AntoineVDV — APPROVED** [review]
@robodoo r+
